# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1317748645 14400
#      Tue Oct 04 13:17:25 2011 -0400
# Node ID 4916b436d0bb9cf86230cbfaa165d94fe41d1dca
# Parent  fba2cc36b762381fd88e8a54cbd94655cd1ec97b
use correct macro to indentify MinGW

* profiler.cc (profile_data_accumulator::query_time):
Use __MINGW32__, not __MINGW__.

diff --git a/src/profiler.cc b/src/profiler.cc
--- a/src/profiler.cc
+++ b/src/profiler.cc
@@ -381,17 +381,17 @@ profile_data_accumulator::get_hierarchic
   return retval;
 }
 
 double
 profile_data_accumulator::query_time (void) const
 {
   octave_time now;
   // FIXME -- this should be removed at some point...  See bug 34210.
-#if defined (__CYGWIN__) || defined (__MINGW__)
+#if defined (__CYGWIN__) || defined (__MINGW32__)
   volatile
 #endif
     double dnow = now.double_value ();
   return dnow;
 }
 
 void
 profile_data_accumulator::add_current_time (void)
